app/models/triannon/annotation.rb

```diff
@@ -103,12 +99,12 @@ def motivated_by
 
     # Add annotation to Solr as a Solr document
     def solr_save
-      solr_writer.write(graph) if id_as_url && !id_as_url.empty?
+      solr_writer.write(graph, root_container) if id_as_url && !id_as_url.empty?
     end
 
     # Delete annotation from Solr
     def solr_delete
-      solr_writer.delete(id) if id
+      solr_writer.delete("#{root_container}/#{id}") if id
```

Comment:
I'm choosing not to do this at this time, but perhaps it would make a good helper method or some such.  Mentally noted for the future.